sdcm/nemesis.py

```diff
@@ -1152,7 +1153,8 @@ def _destroy_data_and_restart_scylla(self, keyspaces_for_destroy: list = None, s
         self.log.debug("Chosen tables: %s", tables)
 
         # Stop scylla service before deleting sstables to avoid partial deletion of files that are under compaction
-        self.target_node.stop_scylla_server(verify_up=False, verify_down=True)
+        with ignore_raft_topology_cmd_failing():
+            self.target_node.stop_scylla_server(verify_up=False, verify_down=True)
 
         try:
```

Comment:
shouldn't this block be inside context?

Reply:
> shouldn't this block be inside context?

I thought the same, but after a short conversation with @aleksbykov, I decided to do everything as is, because once the node is stopped, the cluster will recognize this and won’t send any more requests to it.